source/core_http_client.c

```diff
@@ -1999,7 +1998,7 @@ HTTPStatus_t HTTPClient_Send( const TransportInterface_t * pTransport,
                     "reqBodyBufLen is greater than zero." ) );
         returnStatus = HTTPInvalidParameter;
     }
-    else if( reqBodyBufLen > INT32_MAX )
+    else if( reqBodyBufLen > ( size_t ) ( INT32_MAX ) )
```

Comment:
Why are you including the decimal value of `INT32_MAX` in the log message at line 2005?  This value is well-defined so adding the literal value is not helpful and consumes storage space.

Reply:
Sure thanks